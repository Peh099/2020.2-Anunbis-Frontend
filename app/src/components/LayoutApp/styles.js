import styled from 'styled-components';
import bg_app from "../../assets/images/BG_App.png";

export const Container = styled.div`
    width: 100%;
    min-height: 100%;
    background-image: url(${bg_app});
    display: flex;
    flex-direction: column;
    background-attachment: fixed;
    background-position: center;
    background-repeat: no-repeat;
    background-size: cover;
`

export const Main = styled.main`
    width: fit-content;
    margin-top: 20px;
    display: flex;
    align-items: space-around;
    align-self: center;
    justify-content: center;
`