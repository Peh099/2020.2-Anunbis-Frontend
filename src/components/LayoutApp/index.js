import React from 'react';
import styled from 'styled-components';
import Menu from '../Menu';
import bg_app from "../../assets/BG_App.png";

const Main = styled.main`
    height: 100vh;
    background-image: url(${bg_app});
    display: flex;
    align-items: center;
    justify-content: center;
`

export default function LayoutApp({children}) {
  return (
  <div>
      <header>
         <Menu />
      </header>
      <Main>
          {children}
      </Main>
  </div>
      );
}