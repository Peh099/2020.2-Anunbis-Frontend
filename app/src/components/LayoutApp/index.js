import React from 'react';
import { Container, Main } from './styles';
import Menu from '../Menu';


export default function LayoutApp({ children }) {
    return (
        <Container>
            <Menu />
            <Main>
                {children}
            </Main>
        </Container>
    );
}