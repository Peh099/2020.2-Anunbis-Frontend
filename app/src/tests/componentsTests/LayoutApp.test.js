import { Router } from 'react-router-dom';
import renderer from 'react-test-renderer';
import { createMemoryHistory } from 'history';
import LayoutApp from '../../components/LayoutApp';

describe('Snapshot LayoutApp component', () => {
  it('matches the snapshot', () => {
    const history = createMemoryHistory();
    history.push('/user/professor/');

    const tree = renderer
      .create(
        <Router history={history}>
          <LayoutApp />
        </Router>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
