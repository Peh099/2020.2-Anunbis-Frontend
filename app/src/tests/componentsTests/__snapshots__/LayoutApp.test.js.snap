// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot LayoutApp component matches the snapshot 1`] = `
<div
  className="sc-gtsrHT jftVLd"
>
  <div
    className="sc-gKAaRy hBrgiB"
  >
    <div
      className="sc-iCoGMd ijOiUi"
    >
      <img
        alt="logo"
        className="sc-fujyAs ccinlj"
        onClick={[Function]}
        src="Name_Logo.png"
      />
    </div>
    <div
      className="sc-hKFxyN cLvChq"
    >
      <input
        className="sc-pNWdM gBjkpo"
        onChange={[Function]}
        onKeyPress={[Function]}
        placeholder="Informe o nome do professor"
        type="text"
        width="400px"
      />
    </div>
    <div
      className="sc-jSFjdj gELbaN"
    >
      <button
        className="sc-bdnxRM jBDlKn sc-eCApnc deOVgP"
        onClick={[Function]}
      >
        ººº
      </button>
      
    </div>
  </div>
  <main
    className="sc-dlnjwi itdLzw"
  />
</div>
`;
